connect/src/androidTest/java/com/stripe/android/connect/EmptyEmbeddedComponentActivity.kt

```diff
@@ -17,7 +17,7 @@ class EmptyEmbeddedComponentActivity : FragmentActivity() {
         val title = intent.extras?.getString(KEY_TITLE)
 
         manager = EmbeddedComponentManager(
-            publishableKey = "fake_pk",
+            publishableKey = "pk_test_123",
             clientSecretProvider = { "fake_secret" },
```

Comment:
Doesn't have to be in this PR, but could you also update the client secret to be normally shaped?

Reply:
I had checked in the codebase but didn't see any normal shape for client secrets?